### PR TITLE
Yargs

### PR DESCRIPTION
**Pull request by rgbkrk**
Switch to `yargs`.

### CODE REVIEW

File: src/main/index.js [hunk 1]
```diff
@@ -7,18 +7,13 @@ import { Menu, app } from 'electron';
...
+              .version(version)
+              .argv;
 
-program
-  .version(version)
-  .option('-k, --kernel [kernel]', 'kernel')
-  .parse(process.argv);
-
-const notebooks = program.args;
+const notebooks = argv._;
```
Comment (rgbkrk) [inline src/main/index.js:17]: Note that I tossed the kernel arg since we weren't even using it.

**Comment by jdetle** [comment]
Linter is just wrong in the tests.

**Comment by captainsafia** [comment]
The linter is still complaining about the `console.log` on line 65 in src/main/launch.js.

**Comment by jdetle** [comment]
Yea I was trying to say that that's wrong. It prints out to the terminal just fine.

**Comment by rgbkrk** [comment]
> Yea I was trying to say that that's wrong. It prints out to the terminal just fine.

It does indeed! However, our linting config explicitly forbids the use of `console.log`, causing Travis to return a big ugly ❌. The options to get around this include:
- using `console.warn`
- using `console.error`
- popping up a dialog box using native electron APIs
- using `eslint disable line`

However, all use of `console` should likely be forbidden in the main process as you can only see this when launching nteract from the command line (my comment will make more sense when you're using the built app).

**Comment by jdetle** [comment]
^ Well I guess I'll listen to the linter next time :)

**Comment by jdetle** [comment]
Should we get a third party involved or can I review?

**Comment by rgbkrk** [comment]
:shipit:

**Pull request merged**